File: robocode-tankroyale-bot-api-java/src/main/java/dev/robocode/tankroyale/botapi/events/EventPriority.java
package dev.robocode.tankroyale.botapi.events;

/** Default event priorities. */
public final class EventPriority {
    public static int onWonRound = DefaultEventPriority.ON_WON_ROUND;
    public static int onSkippedTurn = DefaultEventPriority.ON_SKIPPED_TURN;
    public static int onTick = DefaultEventPriority.ON_TICK;
    public static int onCondition = DefaultEventPriority.ON_CONDITION;
//    public static int onTeamMessage = DefaultEventPriority.ON_TEAM_MESSAGE;
    public static int onBotDeath = DefaultEventPriority.ON_BOT_DEATH;
    public static int onBulletFired = DefaultEventPriority.ON_BULLET_FIRED;
    public static int onBulletHitWall = DefaultEventPriority.ON_BULLET_HIT_WALL;
    public static int onBulletHitBullet = DefaultEventPriority.ON_BULLET_HIT_BULLET;
    public static int onBulletHit = DefaultEventPriority.ON_BULLET_HIT;
    public static int onHitByBullet = DefaultEventPriority.ON_HIT_BY_BULLET;
    public static int onHitWall = DefaultEventPriority.ON_HIT_WALL;
    public static int onHitBot = DefaultEventPriority.ON_HIT_BOT;
    public static int onScannedBot = DefaultEventPriority.ON_SCANNED_BOT;
    public static int onDeath = DefaultEventPriority.ON_DEATH;
}

File: robocode-tankroyale-bot-api-csharp/src/events/EventPriority.cs
namespace Robocode.TankRoyale.BotApi.Events
{
  /// <summary>
  /// Effective event priorities. The lower value, the higher priority.
  /// </summary>
  public class EventPriority
  {
    public const int OnWonRound = DefaultEventPriority.OnWonRound;
    public const int OnSkippedTurn = DefaultEventPriority.OnSkippedTurn;
    public const int OnTick = DefaultEventPriority.OnTick;
    public const int OnCondition = DefaultEventPriority.OnCondition;
    //  public const int OnTeamMessage = DefaultEventPriority.OnTeamMessage; // Reserved for future
    public const int OnBotDeath = DefaultEventPriority.OnBotDeath;
    public const int OnBulletFired = DefaultEventPriority.OnBulletFired;
    public const int OnBulletHitWall = DefaultEventPriority.OnBulletHitWall;
    public const int OnBulletHitBullet = DefaultEventPriority.OnHitByBullet;
    public const int OnBulletHit = DefaultEventPriority.OnBulletHit;
    public const int OnHitByBullet = DefaultEventPriority.OnHitByBullet;
    public const int OnHitWall = DefaultEventPriority.OnHitWall;
    public const int OnHitBot = DefaultEventPriority.OnHitBot;
    public const int OnScannedBot = DefaultEventPriority.OnScannedBot;
    public const int OnDeath = DefaultEventPriority.OnDeath;
  }
}